tradingview_ta/main.py

```diff
@@ -97,6 +112,75 @@ def data(symbols, interval, indicators):
 
         return data_json
 
+    def data_multi_intervals(symbols, intervals, indicators):
+        """Format TradingView's Scanner Post Data
+
+        Args:
+            symbols (list): List of EXCHANGE:SYMBOL (ex: ["NASDAQ:AAPL"] or ["BINANCE:BTCUSDT"])
+            interval (list): Time Interval (ex: 1m, 5m, 15m, 1h, 4h, 1d, 1W, 1M)
+
+        Returns:
+            string: JSON object as a string.
+        """
+        output_intervals = []
+        for interval in intervals:
+            if interval == "1m":
```

Comment:
Small suggestion: might be a bit cleaner and slightly more readable if we use match-case syntax here instead of all these ifs and elifs. 
i.e. 
```python
match interval:
    case '1m':
         # 1 Minute
         date_interval = '|1'
         output_intervals.append(data_interval)
    case '5m':
         ...
```

Reply:
}0e52b8646cf72f47390e2e6bc36734630a2c5f8
# match case 526-637
{